README.md

```diff
@@ -1,11 +1,13 @@
 # Scalar Helm Charts Repository
 
-[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0) ![Release Charts](https://github.com/scalar-labs/helm-charts/workflows/Release%20Helm%20Charts/badge.svg?branch=main)
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0) ![Release Charts](https://github.com/scalar-labs/helm-charts/workflows/Release%20Helm%20Charts/badge.svg?branch=main)[![Helm testing and validate for scalar charts](https://github.com/scalar-labs/helm-charts/actions/workflows/helm_charts_scalar.yml/badge.svg)](https://github.com/scalar-labs/helm-charts/actions/workflows/helm_charts_scalar.yml)
```

Comment:
* I think this like can be divided line into three lines, one for each icon. (Currently, the third and second badges are rendered with no gap since there is no space between them.)
* The kind of URLs linked from the second and third icons are different. For the second badge, should the link have been https://github.com/scalar-labs/helm-charts/actions/workflows/release.yml instead of the badge image itself?
* "Helm testing and validate for scalar charts" looks a bit weird and redundant (It's obvious that's for the Helm charts since this is `helm-charts` repo.), and I think it's time to fix it. In my opinion, "Chart Testing and Kubeval" would be clearer and sufficient. (How do you think? @feeblefakie)

Reply:
@ymorimo 
Thank you.
This has been corrected, and we hope you will check it out. :bow:
PTAL
https://github.com/scalar-labs/helm-charts/pull/34/commits/eac9a017ba8536b352384510271e9f7889e1173b

Reply:
@miseyu
How about dividing the line into three for readability, as below?
```
[![License](...
[![Release Charts](...
[![Chart Testing and Kubeval](...
```

And for the badge text, it's needed to change the name in `helm_charts_scalar.yml` as well as the alt text.

Reply:
@ymorimo 
I'm sorry to have bothered you so many times. :bow:
We have corrected the problem. Please check again. :bow:

PTAL
https://github.com/scalar-labs/helm-charts/pull/34/commits/161649824a4614ec947877b117ebad6500f0c861